qucs/qucs/octave_window.cpp

```diff
@@ -119,7 +113,30 @@ bool OctaveWindow::startOctave()
   // append process PATH, othewise Octave does not find gnuplot
   QProcessEnvironment env = QProcessEnvironment::systemEnvironment();
   env.insert("PATH", env.value("PATH") + sep + QucsSettings.BinDir );
+#ifdef __MINGW32__
+  env.insert("PATH", env.value("PATH") + sep + QucsSettings.OctaveBinDir.canonicalPath() );
```

Comment:
New versions of Windows Octave need to load some DLLs from `octave_root/bin` directory. Otherwise Octave failed to start. Windows hasn't an equivalent of `LD_LIBRARY_PATH`. And I haven't found other way than `PATH` modification or changing `octave` process working directory. QProcessEnvironment exists only at process execution time and doesn't modify system environment.

But there may exist another way without version checking. We can simple test `octave-cli` executable. If `octave-cli` exists it will be forced to start instead of `octave`, and `octave` will be started otherwise. I will update my PR later today or tomorrow after checking this.